cookbook/composer.rst

```diff
@@ -14,28 +14,18 @@ following sections.
 Install Composer on Linux and Mac OS X
 --------------------------------------
 
-To install Composer on Linux or Mac OS X, execute the following two commands:
+Run the installer following the Download page instructions from https://getcomposer.org/download.
+
+After this you can run this command to move composer.phar to a directory that is in your path:
 
 .. code-block:: bash
 
-    $ curl -sS https://getcomposer.org/installer | php
     $ sudo mv composer.phar /usr/local/bin/composer
 
-.. note::
-
-    If you don't have ``curl`` installed, you can also just download the
-    ``installer`` file manually at https://getcomposer.org/installer and
-    then run:
-
-    .. code-block:: bash
-
-        $ php installer
-        $ sudo mv composer.phar /usr/local/bin/composer
-
 Install Composer on Windows
 ---------------------------
 
-Download the installer from `getcomposer.org/download`_, execute it and follow
+Download the installer from https://getcomposer.org/Composer-Setup.exe, execute it and follow
```

Comment:
I would change to something like "Download `Composer-Setup.exe`_, execute [...]" and then add the full URL reference at the bottom to have a link that looks nice.